api/src/test/java/org/openmrs/validator/BedTagValidatorTest.java

```diff
@@ -0,0 +1,131 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.validator;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.bedmanagement.entity.BedTag;
+import org.openmrs.module.bedmanagement.service.BedManagementService;
+import org.springframework.validation.BindException;
+import org.springframework.validation.Errors;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Date;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class BedTagValidatorTest {
```

Comment:
In addition to this, we need to a test that ensures the validator is called when `saveBedTag()` is triggered.